### ISSUE TITLE
Fails for some RFCs

### ISSUE THREAD

**Issue by boldt** [jbenner-radham/rfc-to-bib#3]
Here are some RFC, which do not work with the tool:

```
$ rfc-to-bib 768
$ rfc-to-bib 791
$ rfc-to-bib 792
$ rfc-to-bib 793
```

For all of them I get the following error:

**✖ Response code 404 (Not Found)**

**Comment by jbenner-radham** [comment]
Thanks for the bug report! After looking into the issue it appears that for all the BibXML files at https://www.rfc-editor.org/refs/bibxml/ the RFCs are recorded with a 4-digit format. So any RFCs that are not 4-digits are zero padded. So for example the CLI tool is failing because it's looking for https://www.rfc-editor.org/refs/bibxml/reference.RFC.768.xml instead of https://www.rfc-editor.org/refs/bibxml/reference.RFC.0768.xml. Should be an easy fix and I'll put it in the next release. Thanks again for the heads up.

**Comment by jbenner-radham** [comment]
The fix has been published in [v0.2.2](https://github.com/jbenner-radham/rfc-to-bib/tree/0.2.2).
